app-modules/task/src/Actions/UpdateTask.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Assist\Task\Actions;
+
+use Assist\Task\Models\Task;
+use Illuminate\Database\Eloquent\Model;
+
+class UpdateTask
+{
+    public function handle(Task|Model $task, array $data): Task
+    {
+        $data = collect($data);
+
+        /** @var Task $record */
+        $task->fill($data->except('assigned_to')->toArray());
```

Comment:
This does feel a little strange, can we not do this in an observer so we can `Task::update()` like normal?

Reply:
Removed